common/metrics/defs.go

```diff
@@ -709,8 +709,9 @@ const (
 	PersistenceErrBusyCounter
 	PersistenceSampledCounter
 
-	HistoryClientFailures
-	MatchingClientFailures
+	CadenceClientRequests
```

Comment:
Can you also add another tag for client type.  Like 'type=history' for all history client operations and 'type=matching' for all matching client operations?